src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const amount = days * 40;
+  const amountForTheWeek = amount - 50;
```

Comment:
in the previous review's comment it was meant that it is better to put these number(20, 50)s into variables, form example discountWeek, discountThreeDays